matlab_kernel/kernel.json

```diff
@@ -0,0 +1,8 @@
+{
+    "argv": [
+        "python", "-m", "matlab_kernel", "-f", "{connection_file}"],
```

Comment:
Does putting the name "python" here prevent using other versions to start the kernel?

Reply:
It uses the `python` of the environment that launches the kernel.  If another use case is desired, the `python -m matlab_kernel install` method uses the explicit `sys.executable`.